web-ui/src/components/member_selector/member_selector_dialog/__snapshots__/MemberSelectorDialog.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`MemberSelectorDialog > renders correctly 1`] = `
+<div
+  aria-hidden="true"
+/>
```

Comment:
I think we can get this to render with an initialState. Take a look at [this test](https://github.com/objectcomputing/check-ins/blob/5035e949fa13fc491ad446e60f83140dea201a90/web-ui/src/components/edit_skills/EditSkillsCard.test.jsx)

Reply:
I managed to get the test to render the component in the snapshot, but it also outputs a warning:
```
Warning: An update to MemberSelectorDialog inside a test was not wrapped in act(...).
When testing, code that causes React state updates should be wrapped into act(...)
```

I tried a few approaches but couldn't figure out how to get rid of the warning.